### PR TITLE
Renamed 'long_id' to 'purchase_total_long_id'

### PR DESCRIPTION
**Pull request by Joensg**
Renamed '$scope.bb.item_defaults.long_id' to '$scope.bb.item_defaults.purchase_total_id'.

### CODE REVIEW

**Comment by Glenn** [comment]
I'm not sure I like this!
This will break all existing uses. PLus it's not the database id - which is what I would expect this to refer to. Why are we doing this ???

**Comment by lukeferguson** [comment]
The change to add `long_id` was only introduced 19 days ago @Glenn - poor naming that slipped through my initial review - item defaults should be self explanatory for what they represent.

**Comment by Glenn** [comment]
fair enough ;-)

**Comment by lukeferguson** [comment]
I do agree with @Glenn that `purchase_total_id` could be misleading @Joensg - `purchase_total_long_id` would be best!

**Comment by Joensg** [comment]
@lukeferguson ok changed as requested. :)

File: src/core/javascripts/controllers/base.js.coffee [hunk 1]
```diff
@@ -545,8 +545,8 @@ angular.module('BB.Controllers').controller 'BBCtrl', ($scope, $location, $rootS
           $scope.bb.admin = admin
         setup_promises.push sso_admin_login
 
-      if $scope.bb.item_defaults and $scope.bb.item_defaults.long_id
-        total_id = $scope.bb.item_defaults.long_id
+      if $scope.bb.item_defaults and $scope.bb.item_defaults.purchase_total_long_id
```
Comment (WesleyPumpkinhead) [inline src/core/javascripts/controllers/base.js.coffee]: @Joensg it should be purchase_total_id and not purchase_total_long_id

File: src/core/javascripts/controllers/base.js.coffee [hunk 2]
```diff
@@ -545,8 +545,8 @@ angular.module('BB.Controllers').controller 'BBCtrl', ($scope, $location, $rootS
...
+        total_id = $scope.bb.item_defaults.purchase_total_long_id
```
Comment (WesleyPumpkinhead) [inline src/core/javascripts/controllers/base.js.coffee]: @Joensg see above comment

**Pull request merged**